### ISSUE TITLE
CMake fails to find Python library for tests

### ISSUE THREAD

**Issue by merkys** [spglib/spglib#344]
I am building spglib v2.1.0 with CMake on Debian unstable:
```bash
mkdir build && cd build
cmake .. -DSPGLIB_WITH_Fortran=ON -DSPGLIB_WITH_Python=ON
make
make test
```
Tests fail for python-tests with `ModuleNotFoundError: No module named 'spglib'`. I tried setting `PYTHONPATH` to various locations seemingly containing Python files after the build, but I did not succeed.

**Comment by LecrisUT** [comment]
This is an issue to be addressed in #335. Basically the issue is that the python files are not installed in an appropriate location such that the python interpreter can pick it up. In #355 I copy the python source files and inject them in `PYTHONPATH`. The only issue with that PR is that it had surfaced quite a few issues reported by the address sanitization.

Another workaround for this is to use the `pip install` and run `pytest` separately.

If you are building this for packaging to debian, I think it is best to mirror the Fedora packaging https://github.com/spglib/spglib/blob/develop/.distro/spglib.spec . There will be some simplifications and improvements coming from both `spglib` and `scikit-build-core`, and these will be present in tested in the Fedora CI throughout the development process.

**Comment by merkys** [comment]
Thanks for prompt response. OK, I will look into how the packages for Fedora are built.
